docs/qe-available-tasks/sealights/python-instrumentation/python-instrumentation.md

```diff
@@ -0,0 +1,112 @@
+| `target-branch`       | `string` | `main`        | The target branch for the Pull Request (e.g., `main`, `develop`).                             |
+
+## Results
+
+| Name                | Type     | Description                                                                 |
+|---------------------|----------|-----------------------------------------------------------------------------|
+| `sealights-bsid`    | `string` | A unique identifier for the Sealights build session.                       |
+| `sealights-build-name`        | `string` | A unique build name generated using the commit SHA and current date.       |
+| `source-artifact`   | `string` | The URI of the trusted artifact with the application source code.          |
+
+## Volumes
+
+| Name                  | Description                                                    |
+|-----------------------|----------------------------------------------------------------|
+| `sealights-credentials` | Stores Sealights credentials from the specified secret.         |
+| `workdir`             | Temporary working directory for source code operations.        |
+
+## Steps
+
+### 1. `use-trusted-artifact`
+
+Retrieves the source code from a trusted artifact.
+
+### 2. `sealights-python-instrumentation`
+
+Configures and Scans the Python application using Sealights.
+
+## Usage Instructions
+
+### Create the Sealights Secret (REQUIRED)
+
+Ensure you have a Kubernetes secret named **sealights-credentials** containing your Sealights agent token.
+
+We assign the SEALIGHTS_TOKEN var in the script with this command.
+```SEALIGHTS_TOKEN="$(cat /usr/local/sealights-credentials/token)"```
+>NOTE: you must name the value of the secret **token**.
+
+For example:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sealights-credentials
+type: Opaque
+data:
+  token: <BASE64_ENCODED_SEALIGHTS_TOKEN>
+```
+
+### Example Pipeline
+
+Here's an example of how you can use the `sealights-python-instrumentation` task in a Tekton pipelinerun during Konflux CI build.
+
+```yaml
+    - name: sealights-instrumentation
+      runAfter:
+        - clone-repository
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/tekton-integration-catalog.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: tasks/sealights/python-instrumentation/0.1/python-instrumentation.yaml
+      params:
+        - name: source-artifact
+          value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+        - name: python-version
+          value: "311"
+        - name: sealights-secret
+          value: "sealights-credentials"
+        - name: component
+          value: '{{ repo_name }}'
+        - name: branch
+          value: '{{ source_branch }}'
+        - name: revision
+          value: '{{ revision }}'
+        - name: oci-storage
+          value: $(params.output-image).sealights.git
```

Comment:
we probably don't need this parameter anymore right

Reply:
Correct I'll remove it